packages/components/src/segmented-control/segmented-control-backdrop.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect, memo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import type { SegmentedControlBackdropProps } from './types';
+import { BackdropView } from './styles';
+
+function SegmentedControlBackdrop( {
+	containerRef,
+	containerWidth,
+	state,
+}: SegmentedControlBackdropProps ) {
+	const [ left, setLeft ] = useState( 0 );
+	const [ width, setWidth ] = useState( 0 );
+	const [ canAnimate, setCanAnimate ] = useState( false );
+
+	useEffect( () => {
+		const containerNode = containerRef?.current;
+		if ( ! containerNode ) return;
+
+		/**
+		 * Workaround for Reakit
```

Comment:
Maybe we should add some details here for future reference? @diegohaz do you know any additional information about this workaround?

Reply:
It's not clear to me what this workaround is solving either. 🤔